modules/ozoneBidAdapter.js

```diff
@@ -234,38 +240,35 @@ export const spec = {
           obj.ext.ozone.customData = [{'settings': {}, 'targeting': {'oztestmode': ozTestMode}}];
         }
       } else {
-        utils.logInfo('no ozTestMode ');
+        utils.logInfo('OZONE: no ozTestMode ');
```

Comment:
Just a heads up, there have been discussions around the amount of logging bidAdapters are allowed to include. Obviously these are only fired if the user has debug on, but strings cannot be minified. So the prebid package becomes quite large.

For now, no changes are necessary as we have not come up with a plan to actually enforce a logging amount. But wanted to let you know!

Reply:
Noted - we'll look at cleaning up / removing unnecessary logs in a future version or maybe introducing error/logging codes to reduce the length of the strings which should help with keeping string lengths to a minimum.